identity/src/main/java/com/stripe/android/identity/ui/SelfieScreen.kt

```diff
@@ -0,0 +1,292 @@
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Checkbox
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.compose.ui.graphics.painter.BitmapPainter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.semantics.testTag
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.viewinterop.AndroidView
+import com.google.android.material.composethemeadapter.MdcTheme
+import com.stripe.android.camera.framework.image.mirrorHorizontally
+import com.stripe.android.camera.scanui.CameraView
+import com.stripe.android.identity.R
+import com.stripe.android.identity.networking.Resource
+import com.stripe.android.identity.networking.models.VerificationPage
+import com.stripe.android.identity.states.FaceDetectorTransitioner
+import com.stripe.android.identity.states.IdentityScanState
+import com.stripe.android.uicore.text.Html
+import com.stripe.android.uicore.text.dimensionResourceSp
+
+internal const val SELFIE_VIEW_FINDER_ASPECT_RATIO = 1f
+internal const val SELFIE_SCAN_TITLE_TAG = "SelfieScanTitle"
+internal const val SELFIE_SCAN_MESSAGE_TAG = "SelfieScanMessage"
+internal const val SELFIE_SCAN_CONTINUE_BUTTON_TAG = "SelfieScanContinue"
+internal const val SCAN_VIEW_TAG = "SelfieScanViewTag"
+internal const val RESULT_VIEW_TAG = "SelfieResultViewTag"
+internal const val CONSENT_CHECKBOX_TAG = "ConsentCheckboxTag"
+private const val FLASH_MAX_ALPHA = 0.5f
+private const val FLASH_ANIMATION_TIME = 200
+
+@Composable
+internal fun SelfieScanScreen(
+    title: String,
+    message: String,
+    verificationPageState: Resource<VerificationPage>,
+    onError: (Throwable) -> Unit,
+    newDisplayState: IdentityScanState?,
+    onCameraViewCreated: (CameraView) -> Unit,
+    onContinueClicked: (Boolean) -> Unit
+) {
+    MdcTheme {
```

Comment:
Note for the single-activity PR that is coming - not forget to move theme up in the composition hierarchy (instead of in every screen 😄 )

Reply:
yes, will do that! 👍